docs/guides/becoming-a-cna-as-an-open-source-org-or-project.md

```diff
@@ -0,0 +1,186 @@
+
+This guide was written for Open Source organizations and projects that are interested in becoming a CNA and managing their own CVEs. This is not a guide for becoming a [Root CNA](https://www.cve.org/ResourcesSupport/AllResources/CNARules#section_3_roots) which has additional requirements.
+
+This document will help cover the following topics:
+
+* Benefits of becoming a CNA as an Open Source project
+* Does your organization or project meet the requirements to become a CNA?
+* What is the process for becoming a CNA?
+
+A future document will cover other topics about CNA operations.
+
+## Why become a CNA?
+
+CVEs are a common way to record and share details about vulnerabilities in software projects. Because of their near-ubiquity, CVEs are often the method that downstream consumers will use to know when software they're using is vulnerable and requires remediation. This is especially evident with their use of security scanners and tools that report findings discovered in software and systems using the CVE identifiers.
+
+CVE Numbering Authorities (CNAs) are the entities that can allocate CVE IDs and create corresponding CVE Records within a particular scope (a product, a project or group of affiliated projects, and the like). Becoming a CNA means that your organization will have a defined scope and will be able to create CVE IDs and Records for that defined scope autonomously.
+
+Below are some of the benefits of becoming a CNA:
+
+* **Provide high-quality authoritative CVE Records for your users.** CVE Records created by third-parties can be incomplete or inaccurate. 
+* **CVEs can't be issued for projects in a CNA's scope without first reporting to the CNA.** This means that reporters _must_ initially engage with your CNA, thus reducing confusion and allowing subject-matter experts on the project and security policy to weigh in on whether to create a CVE for a given disclosure.
+* **Assign CVE IDs without needing to share embargoed information with other organizations.** This allows the project to determine for themselves  who, if anyone, needs or gets pre-disclosure information.
+
+## Requirements to become a CNA
+
+Before becoming a CNA you can look at the below set of requirements to make sure joining the CNA Program is feasible for your project or organization. **Remember that you can always leave and rejoin the CNA Program at a later date if circumstances change.**
+
+### Points of contact (POCs)
+
+Preferably 2 or more and ordered primary, secondary, etc. For each point of contact you’ll need the following information which is **not publishing publicly** and is only used internally by the CNA Program:
+
+* Name
+* Email address (Can’t be a group email address)
+* Phone number (**Required for primary POC only**, and only used for rare emergencies like [Known Exploited Vulnerabilities](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) in scoped products)
```

Comment:
Does MITRE share the phone numbers of the CNAs with CISA?  If they do, is CISA the only organization they share it with?

Reply:
From @zmanion:

> I'm pretty confident that phone numbers are kept private/internal to the CVE Program, more specifically MITRE in their various Secretariat / TL-Root roles